### PR TITLE
Bugfix/five prime cds mutations

### PR DESCRIPTION
**Pull request by creisle**
BugFixes
- add migration for five prime cds schema change bcgsc/pori_graphkb_parser#5

### CODE REVIEW

File: src/repo/migrate/index.js [hunk 1]
```diff
@@ -557,6 +557,71 @@ const migrate3xFrom13xto14x = async (db) => {
...
+                && variant[breakPoint].pos === 1
+                && variant[breakPoint].offset < 0
+            ) {
+                updated[breakPoint] = { ...variant[breakPoint], offset: 0, pos: variant[breakPoint].offset };
+            }
+        }
+
+        for (const repr of ['break1Repr', 'break2Repr', 'displayName']) {
+            if (variant[repr]) {
+                updated[repr] = variant[repr].replace(/([._])1(-\d+)/g, '$1$2');
```
Comment (elewis2) [inline src/repo/migrate/index.js:615]: what change does this make?

Comment (creisle) [reply src/repo/migrate/index.js:615]: this is for the cds variants that would have been misparsed by the old parser. Previously the parser would have transformed `c.-42G>T` into `c.1-42G>T`. So this corrects that and sets the pos as the offset and fixes the strings

Comment (elewis2) [reply src/repo/migrate/index.js:615]: specifically it's the strings I'm wondering about, it looks like it takes the 1 out ?

Comment (creisle) [reply src/repo/migrate/index.js:615]: yes that's correct. So it puts the `c.1-42G>T` back to what it should be as `c.-42G>T`

Comment (creisle) [reply src/repo/migrate/index.js:615]: You can test this migration by starting the dev server locally and pointing it to either dev or staging db

**Review by elewis2 — APPROVED** [review]

**Pull request merged**